deepchem/models/tests/test_dtnn.py

```diff
@@ -47,3 +49,33 @@ def test_dtnn():
 
     # Check number of parameters
     assert len(list(model.parameters())) == 17
+
+
+@pytest.mark.torch
```

Comment:
We need save/reload/overfit tests. See https://deepchem.readthedocs.io/en/latest/development_guide/coding.html#testing-machine-learning-models